<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>DialogDocumentos</class>
 <widget class="QDialog" name="DialogDocumentos">
  <property name="windowModality">
   <enum>Qt::WindowModal</enum>
  </property>
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>633</width>
    <height>432</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Documentos</string>
  </property>
  <widget class="QWidget" name="gridLayoutWidget_2">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>10</y>
     <width>611</width>
     <height>411</height>
    </rect>
   </property>
   <layout class="QGridLayout" name="gridLayout">
    <item row="2" column="2">
     <widget class="QLineEdit" name="lineEdit_doc_descricao"/>
    </item>
    <item row="4" column="2">
     <layout class="QHBoxLayout" name="horizontalLayout">
      <item>
       <widget class="QTableView" name="tableView"/>
      </item>
     </layout>
    </item>
    <item row="0" column="0">
     <widget class="QLabel" name="label">
      <property name="text">
       <string>Tipo:</string>
      </property>
     </widget>
    </item>
    <item row="0" column="2">
     <widget class="QComboBox" name="comboBox_doc_tipo">
      <property name="sizePolicy">
       <sizepolicy hsizetype="Preferred" vsizetype="Maximum">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
     </widget>
    </item>
    <item row="2" column="0">
     <widget class="QLabel" name="label_3">
      <property name="text">
       <string>Descrição: </string>
      </property>
     </widget>
    </item>
    <item row="3" column="2">
     <layout class="QHBoxLayout" name="horizontalLayout_2">
      <item>
       <widget class="QPushButton" name="pushButton_cancelar">
        <property name="text">
         <string>&amp;Cancelar</string>
        </property>
       </widget>
      </item>
      <item>
       <spacer name="horizontalSpacer">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>40</width>
          <height>20</height>
         </size>
        </property>
       </spacer>
      </item>
      <item>
       <widget class="QPushButton" name="pushButton_inserir">
        <property name="text">
         <string>&amp;Inserir</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item row="1" column="0">
     <widget class="QLabel" name="label_2">
      <property name="text">
       <string>Núm:</string>
      </property>
     </widget>
    </item>
    <item row="1" column="2">
     <widget class="QLineEdit" name="lineEdit_doc_num"/>
    </item>
    <item row="5" column="2">
     <layout class="QHBoxLayout" name="horizontalLayout_3">
      <item>
       <spacer name="horizontalSpacer_2">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>40</width>
          <height>20</height>
         </size>
        </property>
       </spacer>
      </item>
      <item>
       <widget class="QPushButton" name="pushButton_fechar">
        <property name="text">
         <string>&amp;Fechar</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
 </widget>
 <tabstops>
  <tabstop>comboBox_doc_tipo</tabstop>
  <tabstop>lineEdit_doc_num</tabstop>
  <tabstop>lineEdit_doc_descricao</tabstop>
  <tabstop>pushButton_inserir</tabstop>
  <tabstop>pushButton_cancelar</tabstop>
  <tabstop>tableView</tabstop>
  <tabstop>pushButton_fechar</tabstop>
 </tabstops>
 <resources/>
 <connections>
  <connection>
   <sender>pushButton_fechar</sender>
   <signal>clicked()</signal>
   <receiver>DialogDocumentos</receiver>
   <slot>close()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>581</x>
     <y>357</y>
    </hint>
    <hint type="destinationlabel">
     <x>316</x>
     <y>215</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
